api/src/main/resources/application.properties

```diff
@@ -1,8 +1,8 @@
 
-spring.datasource.url=jdbc:postgresql://localhost:5432/amazinDb
+spring.datasource.url=jdbc:postgresql://localhost:5432/amazinDB
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.datasource.username=postgres
-spring.datasource.password=123456
+spring.datasource.password=root
```

Comment:
Please update db connection url to end with amazinDb  <---note the lowercase 'b'. I set it up a second time due to issue i was having and forgot which way I had made it but we need to keep this consistent. 

Also the password I used was 123456.  If we want to change them to something else we will need to let the team know.